File: 0001A.Theatre.Square/solution.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Codeforces
{
    class Program
    {
        static void Solution()
        {
            List<UInt64> line = System.Console.ReadLine().Split().Select(UInt64.Parse).ToList();
            UInt64 height = line[0], width = line[1], size = line[2];

            UInt64 height_ratio = (height % size == 0) ? (height / size) : ((height / size) + 1);
            UInt64 width_ratio = (width % size == 0) ? (width / size) : ((width / size) + 1);

            System.Console.WriteLine("{0}", height_ratio * width_ratio);
        }

        static void Main(string[] args)
        {
            Solution();
        }
    }
}

File: 0001A.Theatre.Square/solution.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

var reader *bufio.Reader = bufio.NewReader(os.Stdin)
var writer *bufio.Writer = bufio.NewWriter(os.Stdout)

func Printf(format string, x ...interface{}) {
	fmt.Fprintf(writer, format, x...)
}

func Scanf(format string, x ...interface{}) {
	fmt.Fscanf(reader, format, x...)
}

func Solution() {
	var height, width, size uint64
	Scanf("%d %d %d\n", &height, &width, &size)

	var heightRatio uint64
	if height%size == 0 {
		heightRatio = height / size
	} else {
		heightRatio = height/size + 1
	}

	var widthRatio uint64
	if width%size == 0 {
		widthRatio = width / size
	} else {
		widthRatio = width/size + 1
	}

	Printf("%d\n", heightRatio*widthRatio)
}

func main() {
	Solution()
	writer.Flush()
}
